### PR TITLE
Typo in eip-191

### PR DESCRIPTION
**Pull request by DrakeEvans**
I noticed a typo in eip-191, so I thought I would fix it.  I couldn't find any contributing guidelines so I hope this is ok.

### CODE REVIEW

**Review by MicahZoltu — APPROVED** [review]

File: EIPS/eip-191.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Several multisignature wallet implementations have been created which accepts `p
 * Multisignature wallets have also had the problem that a `presigned` transaction has not been tied to a particular `validator`, i.e a specific wallet. Example:
     1. Users `A`, `B` and `C` have the `2/3`-wallet `X`
     2. Users `A`, `B` and `D` have the `2/3`-wallet `Y`
-    3. User `A` and `B` submites `presigned` transaction to `X`.
+    3. User `A` and `B` submits `presigned` transaction to `X`.
```
Comment (MicahZoltu) [inline EIPS/eip-191.md:24]:
```suggestion
    3. User `A` and `B` submit `presigned` transactions to `X`.
```

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
Thinking twice on this, if we are going to fix the typo, we should fix it all the way.  😄

**Review by holiman — APPROVED** [review]

**Comment by alita-moore** [comment]
Did this not auto merge @MicahZoltu?

**Comment by alita-moore** [comment]
oh @MicahZoltu this wasn't auto merging because you had requested changes.

**Comment by alita-moore** [comment]
so you probably needed to approve it again

**Comment by MicahZoltu** [comment]
> so you probably needed to approve it again

Ah, my bad.  Thanks for noticing, I'll try to keep my eye out for this in the future.

**Pull request merged**